core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -1326,6 +1330,17 @@ public static MeterRegistry meterRegistry() {
         return METER_REGISTRY;
     }
 
+    /**
+     * Returns the default interval in milliseconds between reporting unhandled exceptions by armeria.
+     *
+     * <p>The default value of this flag is
+     * {@link DefaultFlagsProvider#DEFAULT_UNHANDLED_EXCEPTIONS_REPORT_INTERVAL_MILLIS}.</p>
```

Comment:
Can we also mention the JVM option for overriding this like other flags?

Reply:
Sure, that's much better because I also had hard time figuring out how to configure that JVM flag 😅 .